plugins/module_utils/network/junos/facts/facts.py

```diff
@@ -140,9 +144,7 @@ class Facts(FactsBase):
     def __init__(self, module):
         super(Facts, self).__init__(module)
 
-    def get_facts(
```

Comment:
we don;t need to change the formatting here

Reply:
the formatting changes are introduced due to black. The default line length is 88 characters. Is that not the default formatter used?